Specs/Number/English/NumberModel.json

```diff
@@ -1634,4 +1634,79 @@
   "NotSupportedByDesign": "python",
   "Results": []
 }
+,
+{
+  "Input": "The answer is negative ninety - five hundred fifths",
+  "NotSupportedByDesign": "python",
+  "NotSupported": "javascript",
+  "Results": [
+    {
+      "Text": "negative ninety - five hundred fifths",
+      "TypeName": "number",
+      "Resolution": {
+        "value": "-1900"
+      }
+    }
+  ]
+}
+,
+{
```

Comment:
Can you add more tests cases like "minus 5", "minus five percent", "minus five point five" and similar?